### PR TITLE
Ship CRO | .io Homepage Hero Test #2 for all relevant .io homepages

### PR DESCRIPTION
**Pull request by alexcarpenter**
## 🔗 Relevant links

- [Preview link](https://app.asana.com/0/1202791227659279/1202991873933648/f) 🔎
- [Asana task](https://dev-portal-git-aclaunch-cro-hero-alt-experiment-hashicorp.vercel.app/) 🎟️

## 🗒️ What

Step 1 to ship homepage hero test to related homepages.

## 📸 Design Screenshots

**Vault:**
CleanShot 2023-02-21 at 14 37 55@2x

**Consul:**
CleanShot 2023-02-21 at 14 38 33@2x

**Boundary**
CleanShot 2023-02-21 at 14 53 09@2x

**Packer**
CleanShot 2023-02-21 at 14 53 31@2x

## 🧪 Testing

- [ ] Navigate to vault io homepage and alt hero is rendered as in design screenshots section
- [ ] Navigate to consul io homepage and alt hero is rendered as in design screenshots section
- [ ] Navigate to packer io homepage and alt hero is rendered as in design screenshots section
- [ ] Navigate to boundary io homepage and alt hero is rendered as in design screenshots section

### CODE REVIEW

File: src/middleware.ts [hunk 1]
```diff
@@ -109,38 +109,18 @@ export async function middleware(req: NextRequest, ev: NextFetchEvent) {
...
+	// 	['vault', 'consul'].includes(product) &&
+	// 	['/'].includes(req.nextUrl.pathname)
+	// ) {
+	// 	try {
+	// 		const edgeFlags = await getEdgeFlags({ request: req })
+	// 		const { flags, cookie } = edgeFlags
+	// 	} catch {
+	// 		// Fallback to default URLs
+	// 	}
+	// }
```
Comment (alexcarpenter) [inline src/middleware.ts:123]: Commenting out because we have a quick follow up experiment within the io homepage heros.

**Comment by alexcarpenter** [comment]
Terraform PR: https://github.com/hashicorp/terraform-website/pull/2468

**Review by dstaley — APPROVED** [review]
I don't think Dev Portal is optimizing SVGs anymore, so we need to run these through something like https://jakearchibald.github.io/svgomg/. Looks like we can shave off ~60% of their size.

Everything else looks great!

**Comment by alexcarpenter** [comment]
@dstaley hm, I did run them through svgo. Where are you seeing 60% savings on?

**Comment by dstaley** [comment]
I ran `/boundary-public/img/hero-pattern-end.svg` through with default settings. Maybe svgomg is more agressive than svgo's defaults?

**Comment by alexcarpenter** [comment]
@dstaley ahh, I must have made some changes to the settings which wasn't optimizing as much. Reset back to defaults and seeing the savings now! Good catch.

**Comment by dstaley** [comment]
As a future improvement we could probably configure the io-hero-alt component to accept a `mirror` prop that uses the same image source but mirrors one side. I like the flexibility of being able to specify two completely different images, but if all of these heroes are going to use mirrored patterns I think it's worth implementing.

**Pull request merged**